topics/statistics/tutorials/galaxy-ludwig/tutorial.md

```diff
@@ -0,0 +1,455 @@
+
+## Combined Loss
+
+The Loss vs. Epoch graph represents the learning progress during the training of a neural network. The x-axis shows the epochs, where one epoch means that each sample in the training dataset has had an opportunity to update the model’s internal parameters. The y-axis represents the loss, a numerical metric indicating how incorrect the model's predictions are. The objective of training is to minimize this loss, aiming to reduce it to the lowest possible value.
+
+> <tip-title>Loss Function details</tip-title>
+>In image classification tasks, cross-entropy loss, also known as log loss, is the most frequently used loss function. For binary classification (two classes), binary cross-entropy is used, while sparse categorical cross-entropy is applied when classifying three or more categories. The model produces a probability vector indicating the likelihood that the input image belongs to each predefined category. This predicted vector is then compared to the actual output, which is represented by a vector of the same size, where the correct category is assigned a probability of 1, and all other categories are assigned a probability of 0.
+>
+{: .tip}
+
+In the graph (Figure 9), there is a training curve. When training a model, you'll often look at a loss curve to determine if the model has converged. It could also help in detecting overfitting and underfitting and can provide insights for implementing early stopping.
+
+> <tip-title>Overfitting</tip-title>
+>Overfitting occurs when the network fails to generalize well to unseen data because it begins to model the noise and randomness in the training dataset.
+>![alt](../../images/galaxy-ludwig/explain_overfitting.png "Overfitting curve")
+>
+{: .tip}
+
+> <tip-title>Underfitting</tip-title>
+>Underfitting happens when the patterns in the training dataset are too complex for our model. It can’t capture them, so it has a poor performance on training and validation sets. 
+>![alt](../../images/galaxy-ludwig/explain_underfitting.png "Underfitting curve")
+>
+{: .tip}
+
+![loss vs epochs](../../images/galaxy-ludwig/test1_combined_loss.png "Loss versus Epochs during training. The graph illustrates the relationship between the training loss and the number of epochs during the training of the model.")
+
+Based on the training curve (blue line), we can draw the following conclusions:
+
+- CONSISTENT IMPROVEMENT:
+    Training loss consistently decrease over the five epochs. This indicates that the model is effectively learning and improving its performance on the dataset.
+- APPROACHING CONVERGENCE:
+    By Epoch 5, the training loss has reduced to less than 0.5. The gradual reduction in losses suggests that the model is approaching convergence, where further training may yield diminishing returns in terms of performance improvement.
```

Comment:
Do we know something about the `test loss`? Observing test/validation loss is also important to decide whether a model is generalising well or not

Reply:
Yes, we have this information. 
It is present in the report file under predictions.shapes. 
It is not included in the graph, which is why it isn’t mentioned directly; the focus was to provide a brief explanation of the graphs.

Reply:
I added an additional information point to highlight this result and its conclusion.